src/ucs/arch/x86_64/cpu.c

```diff
@@ -24,6 +28,9 @@ static inline void ucs_x86_cpuid(uint32_t level, uint32_t *a, uint32_t *b,
                   : "0" (level));
 }
 
+#define xgetbv(index, eax, edx) \
+	asm volatile (".byte 0x0f, 0x01, 0xd0" : "=a"(eax), "=d"(edx) : "c" (index))
```

Comment:
can you please document this reason in the code (and possibly providing a reference link)?